server/db/index.js

```diff
@@ -15,37 +15,40 @@ function getRandom(type, max) {
 }
 
 const syncAndSeed = () => {
-  return conn
-    .sync({ force: true })
-    .then(() => {
-      return Promise.all([
-        Promise.all(usersSeed.map(user => User.create(user))),
-        Promise.all(challengesSeed.map(chal => Challenge.create(chal))),
-        Promise.all(userchallengeSeed.map(chal => Userchallenge.create(chal))),
-        Promise.all(solutionsSeed.map(sol => Solution.create(sol))),
-        Promise.all(
-          imagesSeed.map(img => Image.create({ connector: img.connector, data: img.data })),
-        ),
-      ]);
-    })
-    .then(([users, challenges, userchallenges, solutions, images]) => {
-      return Promise.all([
-        userchallenges
-          .find(chal => chal.css.includes('circle'))
-          .update({ userId: getRandom(users, 3), challengeId: 1 }),
-        userchallenges
-          .find(chal => chal.css.includes('square'))
-          .update({ userId: getRandom(users, 3), challengeId: 2 }),
-        solutions.find(sol => sol.css.includes('circle')).update({ challengeId: 1 }),
-        solutions.find(sol => sol.css.includes('square')).update({ challengeId: 2 }),
+  return (
+    conn
+      .sync({ force: true })
+      .then(() => {
+        return Promise.all([
```

Comment:
Nice.

Reply:
I'd love to take credit for this but this was all Theo's work, I just changed the indentation